gc/base/MemoryPoolLargeObjects.cpp

```diff
@@ -355,6 +358,8 @@ MM_MemoryPoolLargeObjects::calculateTargetLOARatio(MM_EnvironmentBase* env, uint
 
 	_minLOAFreeRatio = *std::min_element(_loaFreeRatioHistory, _loaFreeRatioHistory + _extensions->loaFreeHistorySize);
 
+	Assert_GC_true_with_message2(env, ((0 <= _minLOAFreeRatio) && (1.0 >= _minLOAFreeRatio)), "minLOAFreeRatio(%zu) should not be between 0 and 1.0, _loaFreeRatioHistory[0]=%f.\n", _minLOAFreeRatio, _loaFreeRatioHistory[0]);
+
```

Comment:
Add one assert above, as soon as we fetch loaFreeBytes that it's <= _loaSize

Reply:
printing just first element of the array is not that useful. if this really asserts and we get core, we can examine the array.

Reply:
I don't think 'new line' at the end of message is needed.